website/routes.py

```diff
@@ -1487,6 +1488,45 @@ def make_url_map(app):
             json_renderer,
         ),
 
+        Rule(
+            [
+                '/project/<pid>/mailing_list/',
+                '/project/<pid>/node/<nid>/mailing_list/',
+            ],
+            'get',
+            mailing_list_views.get_node_mailing_list,
+            json_renderer,
+        ),
```

Comment:
Wrote a follow-up ticket to migrate these to v2: https://openscience.atlassian.net/browse/OSF-6160